pages/index.css

```diff
@@ -21,3 +21,19 @@
 
 /* Visually Hidden Mix */
 @import url(../blocks/visually-hidden/visually-hidden.css);
+
+/*Block Footer*/
```

Comment:
@ElenaPodv, could you please move your blocks' imports above the Visually Hidden Mix import? This will help prevent unintended style overwrites and improve the flow of cascading styles.

Reply:
Done